### ISSUE TITLE
Add gold and copper coins for trading

### ISSUE THREAD

**Issue by MarkuBu** [D00Med/farlands#147]
I think, gold ingots are a bit to much for trading. I would suggest to add gold and copper coins. 

One ingot = 9 coins
10 copper coins = 1 gold coin

Coins can be found in chests. Or you can make coins from nuggets.

**Comment by tobyplowy** [comment]
@MarkuBu yes good idea but I can't program so if you want them then you will have to add them :/

**Comment by MarkuBu** [comment]
Currently I just drop some thoughts and want to know your opinion. I will work on the farming mod first, but I may make minor changes in between

**Comment by tobyplowy** [comment]
@MarkuBu are you interested in adding coins

If you say yes than I will begin making the textures :)

**Comment by MarkuBu** [comment]
What does @D00Med say? If it is ok, keep going

**Comment by tobyplowy** [comment]
@MarkuBu d00med said that it was a good idea (but that was a long time ago) 
He may have changed his mind by now
@D00Med guess you will have to say it :)

**Comment by MarkuBu** [comment]
I had the idea that you can't craft coins yourself, what is some kind of logical. 

There are two ways to get coins. Find them in loot chests or trade by a dealer. You get 10 coins for one ingot

**Comment by tobyplowy** [comment]
@MarkuBu maybe that can only count for gold coins 

but maybe silver and copper coins can be gained if you kill agresive mobs you need a 100 copper coins for one silver coin and a 100 silver coins for one gold coin

Maybe we should also call coins bits it sounds nicer :)

Is that a good idea?

**Comment by MarkuBu** [comment]
Don't make it too complicated

**Comment by tobyplowy** [comment]
@MarkuBu ok :)
